raiden/ui/cli.py

```diff
@@ -552,10 +552,9 @@ def _run_smoketest():
 
         raiden_stdout = StringIO()
         with contextlib.redirect_stdout(raiden_stdout):
-            try:
-                # invoke the raiden app
-                app = run_app(**args)
+            app = run_app(**args)
```

Comment:
Because the `finally` uses `app`, if the call to `run_app` fails the variable `app` will not be set and the `finally` block will raise an exception while handling and exception.

Reply:
Aha I see. That makes sense. Thanks.